docs/prompt_engineering/how_to_guides/multimodal_content.mdx

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 7
+---
+
+# Include multimodal content
+
+Some applications are based around multimodal content, like a chatbot that can answer questions about a PDF or image.
+In these cases, you'll want to include multimodal content in your prompt and test the model's ability to answer questions about the content.
+
+There are two ways to include multimodal content in a prompt:
```

Comment:
any diff between the two that we should call out? how does a user know which approach to pick?